torch/_library/custom_ops.py

```diff
@@ -378,6 +399,26 @@ def fake_impl(*args, **kwargs):
         autograd_impl = _library.autograd.make_autograd_impl(self)
         lib.impl(self._name, autograd_impl, "Autograd")
 
+        schema = self._opoverload._schema
+        if schema.is_mutable:
+
+            def adinplaceorview_impl(*args, **kwargs):
+                for arg, val in _library.utils.zip_schema(schema, args, kwargs):
+                    if not arg.alias_info:
+                        continue
+                    if not arg.alias_info.is_write:
+                        continue
+                    if isinstance(val, Tensor):
+                        autograd.graph.increment_version(val)
+                    elif isinstance(val, (tuple, list)):
```

Comment:
Don't we ban TensorList?

Reply:
TensorList banned in autograd but not for other custom ops.